www/_includes/layouts/components/nav.njk

```diff
@@ -1,37 +1,36 @@
-<div class="container">
-  <nav class="grid-nav">
-    <svg id="toc-drawer-button" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="bars" class="svg-inline--fa fa-bars fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 448 512">
-      <path fill="currentColor" d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"></path>
+<nav class="snow-nav">
+  <button id="toc-drawer-button" class="snow-nav-mobile-open" type="button" aria-expanded="false" aria-controls="nav-primary">
```

Comment:
We were using an `<svg>` button for the nav, which is neither semantic nor accessible. This switches it out for a `<button>` instead, and also adds `aria-*` attributes to help screen readers a bit (we could be doing a lot more, but this is at least a start)